src/mtype.d

```diff
@@ -4747,7 +4747,7 @@ extern (C++) final class TypeSArray : TypeArray
             if (d1 != d2)
             {
             Loverflow:
-                error(loc, "%s size %llu * %llu exceeds 16MiB size limit for static array", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1);
+                error(loc, "Static array size too big", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1);
```

Comment:
I'd remove the `overflow` variable, otherwise lgtm. Just one line of code identical to something nested below, that's not "code duplication".